README.md

```
@@ -39,97 +39,80 @@ Provide an inclusive, multilingual platform that:
+    subgraph Data_Layer [Storage & External]
+        F --> H[(State JSON DB)]
+        F --> I[Google Maps API]
+        F --> J[Places Repo]
+    end
+
+    style Client_Side fill:#e1f5fe,stroke:#01579b
+    style Server_Side fill:#e8f5e9,stroke:#2e7d32
+    style Data_Layer fill:#fff3e0,stroke:#ef6c00
+```
  
+  <img src="https://user-images.githubusercontent.com/74038190/212284100-561aa473-3905-4a80-b561-0d28506553ee.gif" width="100%">
```

markdownlint-cli2 (0.20.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)